homeassistant/components/simplisafe/alarm_control_panel.py

```diff
@@ -37,11 +45,24 @@ class SimpliSafeAlarm(alarm.AlarmControlPanel):
     def __init__(self, system, code):
         """Initialize the SimpliSafe alarm."""
         self._async_unsub_dispatcher_connect = None
-        self._attrs = {}
         self._code = code
         self._system = system
         self._state = None
 
+        self._attrs = {
+            ATTR_SERIAL: system.serial,
+            ATTR_SYSTEM_ID: system.system_id,
+            ATTR_VERSION: system.version,
```

Comment:
These constant values should not be state attributes. They could go into device info.

State attributes should only contain dynamic info related to the state of the entity, or at least something that would be useful in automations.

Reply:
@MartinHjelmare Apologies that this PR is going to get bigger with these changes; totally agree with the request, just an FYI.